core/src/main/java/com/kin/ecosystem/core/data/order/OutgoingTransferCall.java

```diff
@@ -0,0 +1,139 @@
+package com.kin.ecosystem.core.data.order;
+
+
+import android.support.annotation.NonNull;
+
+import com.kin.ecosystem.common.KinCallback;
+import com.kin.ecosystem.common.Observer;
+import com.kin.ecosystem.common.exception.KinEcosystemException;
+import com.kin.ecosystem.core.data.blockchain.BlockchainSource;
+import com.kin.ecosystem.core.data.blockchain.BlockchainSource.SignTransactionListener;
+import com.kin.ecosystem.core.data.blockchain.BlockchainSourceImpl;
+import com.kin.ecosystem.core.data.blockchain.Payment;
+import com.kin.ecosystem.core.network.ApiException;
+import com.kin.ecosystem.core.network.model.OpenOrder;
+import com.kin.ecosystem.core.network.model.Order;
+import com.kin.ecosystem.core.network.model.OutgoingTransferRequest;
+
+import java.math.BigDecimal;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import kin.sdk.migration.common.exception.OperationFailedException;
+
+public class OutgoingTransferCall extends Thread {
+
+    private static final int SSE_TIMEOUT = 30 * 1000; // 30 seconds
```

Comment:
why 30sec? who decided this?

Reply:
after 30 sec on the client side we show to the user - that the transfer is delayed check balance later - no need to listen to block chain for response we will notify the user any more

Reply:
consult with Hadar and Orad to change the timeout to 15 sec

Reply:
change to 15 sec